### PR TITLE
3047: remove minilog dependency and remove JSONPath warning

### PR DESCRIPTION
**Pull request by symn**
This workaround removes `minilog` from dependencies but uses the available browser shim wherever it gets `require`d.
This allows us to use the library e.g within our REST client.

See:
https://github.com/traverson/traverson/issues/289

### CODE REVIEW

File: traverson.js [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
   , errors = errorModule.errors
   , createError = errorModule.createError
   , mediaTypes = require('./lib/media_types')
   , Builder = require('./lib/builder')
   , mediaTypeRegistry = require('./lib/media_type_registry');
 
-// activate this line to enable logging
-if (process.env.TRAVERSON_LOGGING) {
-  require('minilog').enable();
-}
```
Comment (amannn) [inline traverson.js:14]: What's the motivation for this change? The shim also includes an `enable` function, right?

**Review by amannn — COMMENTED** [review]
Great that this is possible with a minimal change! There's only one minor inline question.

Maybe you could propose this PR upstream after merge?

File: traverson.js (hunk 1 above)
Comment (symn) [reply traverson.js:14]: right overlooked this

**Review by amannn — APPROVED** [review]
